src/index.js

```diff
@@ -5,7 +5,7 @@ export default (str, delimiter = '---') => {
 
   str = str.trim()
 
-  const RE = new RegExp(`^${delimiter}\\n+([\\s\\S]+)\\n+${delimiter}(?:\\n([\\s\\S]*))?$`)
+  const RE = new RegExp(`^${delimiter}[\\r|\\n]+([\\s\\S]+)[\\r|\\n]+${delimiter}(?:[\\r|\\n]([\\s\\S]*))?$`)
```

Comment:
maybe `\\r?\\n`?

Reply:
I'm rather new to regex, I'll review it again later this night... I have tested the code using the unit testing (jest) available on the repo and it work just fine

Reply:
After doing some testing, i've found the correct regex string would be `(\\r|\\n)+` or `(\\[r?n])+`. They will produce an identical output. Which do you prefer?

Reply:
After doing some testing, i've found the correct regex string would be `(\\r|\\n)+` or `(\\[r?n])+`. They will produce an identical output. Which one do you prefer?

Reply:
<img width="288" alt="2018-10-15 1 55 26" src="https://user-images.githubusercontent.com/8784712/46932388-14e9e980-d082-11e8-93ff-8ce4fda77a67.png">